src/Sqs/Result/SendMessageResultTrait.php

```diff
@@ -10,8 +10,6 @@ protected function populateFromResponse(ResponseInterface $response): void
     {
         $data = new \SimpleXMLElement($response->getContent(false));
 
-        // TODO Verify correctness
-        $data = $data->SendMessageResult;
```

Comment:
this should have been generated by the `wrapper` thing

Reply:
Thank you!